### ISSUE TITLE
Power Of Two Integers 

### ISSUE THREAD

**Issue by AnkitaKataria22** [makesmatheasy/makesmatheasy#5241]

**Comment by AnkitaKataria22** [comment]
Given a positive integer which fits in a 32 bit signed integer, find if it can be expressed as A^P where P > 1 and A > 0. A and P both should be integers.

Example

Input : 4
Output : True  
as 2^2 = 4

**Comment by AnkitaKataria22** [comment]
pls assign me this issue

**Comment by sairish2001** [comment]
Power function is already there in the Calculator,
if you can design your own logic, and provide steps to it then please tell

**Comment by sairish2001** [comment]
Like if user enters 2^3

then steps should be like

2×2×2
4×2
8
something like this

**Comment by AnkitaKataria22** [comment]
> Power function is already there in the Calculator,
> if you can design your own logic, and provide steps to it then please tell
yes I wanna solve this issue by using log function

**Comment by sairish2001** [comment]
> 
> yes I wanna solve this issue by using log function

Okay, no matter how you solve, but be make sure to show steps for it. As shown above. 
If you feel any difficulty while solving issue or adding steps do tell us, we will help you with that.
for Steps we are using LaTex text, it's documentation is in the slack channel

**Comment by jatinjain001** [comment]
i would like to work kindly assign it to me please

**Comment by sairish2001** [comment]
Go ahead
Deadline 22 June,2021

**Comment by jatinjain001** [comment]
as there is already the power function in the calulator then what is the issue clarify me ,

**Comment by sairish2001** [comment]
> Like if user enters 2^3
> 
> then steps should be like
> 
> 2×2×2
> 4×2
> 8
> something like this
> 

@jatinjain001

**Comment by sairish2001** [comment]
make different function for it

**Comment by jatinjain001** [comment]
Okay

**Comment by Ayushkumarsinghyogesh** [comment]
assign me this issue

**Comment by arpit456jain** [comment]
@Ayushkumarsinghyogesh please wait for 2 days if @jatinjain001 is unable to make PR then i will assign it to you :)
Meanwhile explore other issue , mention me or PA in the comment and it it will be assigned to you .

**Comment by arpit456jain** [comment]
@jatinjain001 any updates?

**Comment by jatinjain001** [comment]
I was playing  a hackathon and hackathon ends today , Basically I changes
in the UI but I am trying to work on logic now

**Comment by arpit456jain** [comment]
@jatinjain001 any updates ? its been a long time

**Comment by jatinjain001** [comment]
No update at all  cause  semester exams issue  ..

**Comment by Sin-Sumit** [comment]
Hey, Where it is mentioned?

**Comment by arpit456jain** [comment]
@Ayushkumarsinghyogesh are you interested?

**Comment by Sin-Sumit** [comment]
First please tell the section where I get this and then I figure it out whether I can do it or not.

**Comment by Ayushkumarsinghyogesh** [comment]
yes i'm interested , but i couldn't cz of my exams which will probably over
after a week. can i do it after my exams?

**Comment by arpit456jain** [comment]
> First please tell the section where I get this and then I figure it out whether I can do it or not.

clone the project and then open it using live server of vs code you will find the section

**Comment by Sin-Sumit** [comment]
Hey @arpit456jain I'm unable to find the powertopower file. Please tell me under which section it is.
